public/img/search.svg

```diff
@@ -0,0 +1,5 @@
+<svg enable-background="new 0 0 385 385" version="1.1" viewBox="0 0 385 385" xml:space="preserve" xmlns="http://www.w3.org/2000/svg">
```

Comment:
I recommend using this cool tool to minify the SVG - https://jakearchibald.github.io/svgomg/ without changing any visuals.

This is my setup, quite random, so you can surely improve:

https://ibb.co/7tnSLbK

Reply:
Thank you, I will use it. I had some cool thing for minification in console, but I cant remember what was it. 🙈